### DISCUSSION TITLE
How to contribute to the project ?

### DISCUSSION THREAD

**Discussion by noor-tg** [feathersjs/feathers#2689] [Q&A] [answered]
hi, I want to contribute to the project . is there any guide or roadmap of features list ? so I can contribute ?

**Comment by daffl** [comment, marked as answer]
We are currently finalising v5 so the best contributions are testing out the guides and reviewing the API docs at https://dove.feathersjs.com. There is a lot in flight so the best way to get started would be to [join our Discord](https://discord.com/invite/qa8kez8QBx) for further discussion.

**Reply by noor-tg** [reply]
any help with the code ?

**Reply by daffl** [reply]
Essentially any of the [open issues](https://github.com/feathersjs/feathers/issues) but some do require a more in depth knowledge of the codebase. As I mentioned, Discord (at the moment specifically the #_dove channel) is the best place to start figuring out what is going on and could use help.
